packages/db/drizzle/0000_overconfident_molten_man.sql

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE `incident` (
+	`id` int AUTO_INCREMENT PRIMARY KEY NOT NULL,
+	`status` enum('resolved','investigatin',''),
+	`page_id` int,
+	`created_at` timestamp NOT NULL DEFAULT (now()),
+	`updated_at` timestamp NOT NULL ON UPDATE CURRENT_TIMESTAMP);
+--> statement-breakpoint
+CREATE TABLE `incidentUpdate` (
+	`id` int AUTO_INCREMENT PRIMARY KEY NOT NULL,
+	`incident_date` datetime,
+	`title` varchar(256),
+	`message` text,
+	`incident_id` int NOT NULL,
+	`created_at` timestamp NOT NULL DEFAULT (now()),
+	`updated_at` timestamp NOT NULL ON UPDATE CURRENT_TIMESTAMP);
```

Comment:
What is the purpose of the table?

We will have to store every single change of an incident. Could be interesting to use tinybird for that.

Reply:
An incident in my mind was a set message you post when your service is down! 
and the message is this table, an update to an incident